tests/framework/backup.go

```diff
@@ -230,7 +230,8 @@ func WaitForBackupPhase(ctx context.Context, backupName string, backupNamespace
 	err := wait.PollImmediate(pollInterval, waitTime, func() (bool, error) {
 		backup, err := GetBackup(ctx, backupName, backupNamespace)
 		if err != nil {
-			return false, err
+			ginkgo.By(fmt.Sprintf("Failed getting backup: %s", err.Error()))
```

Comment:
do we know why this may fail?

Reply:
Its calling veleroCLI get backup command and does some other functionalities in the Getbackup function. I assume it can fail..